AY2021S2-CS2103-T14-1/tp
# Update DG diagram and storage interface

### What this does 
Closes #315 

### How to test
- Check if the diagrams have solid lines with arrows for the association.
- Check if `Storage` has all the correct methods.
